public_suffix_list.dat

```diff
@@ -14860,11 +14860,39 @@ from.tv
 sakura.tv
 
 // Salesforce.com, Inc. https://salesforce.com/
-// Submitted by Michael Biven <mbiven@salesforce.com> and Aaron Romeo <aaron.romeo@salesforce.com>
+// Submitted by Michael Biven <mbiven@salesforce.com>, Aaron Romeo <aaron.romeo@salesforce.com>, and Steven Lawrance <slawrance@salesforce.com>
```

Comment:
Please replace this with a single non-personal e-mail.

Reply:
@simon-friedberger Thanks I now have that done in the newest commit, where those colleagues and I are collapsed into public-suffix-list@salesforce.com.